pages/pipelines/group_step.md.erb

```diff
@@ -0,0 +1,213 @@
+
+<section class="Docs__note">
+  <h3>Note</h3>
+  <p>Sub-groups and nested groups are not supported.</p>
+</section>  
+
+The group step type also groups them up by a `group_identifier` if it exists.
+
+## Group step attributes  
+
+_Required attributes:_  
+
+<table data-attributes>
+<tr>
+    <td><code>group</code></td>
+    <td>
+    Name of the group in the UI. In YAML, if you don't want a label, pass a `~`. Can also be provided in the `label` attribute if `null` is provided to the `group` attribute.<br>
+      <em>Type:</em> <code>string</code> or <code>null</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>steps</code></td>
+    <td>
+    A list of steps in the group; at least 1 step is required. Allowed step types: `wait` and `trigger`.<br>
+      <em>Type:</em> <code>array</code>
+    </td>
+  </tr>
+</table>
+
+_Optional attributes:_  
+
+<table data-attributes>
+  <tr>
+    <td><code>allow_dependency_failure</code></td>
+    <td>
+      Whether to continue to run this step if any of the steps named in the <code>depends_on</code> attribute fail.<br>
+      <em>Default:</em> <code>false</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>branches</code></td>
+    <td>
+      The <a href="/docs/pipelines/branch-configuration#branch-pattern-examples">branch pattern</a> defining which branches will include this step in their builds.<br>
+      <em>Example:</em> <code>"main stable/*"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>depends_on</code></td>
+    <td>
+      A list of step keys that this step depends on. This group will only run after the named steps have completed. See <a href="/docs/pipelines/dependencies">managing step dependencies</a> for more information.<br>
+      <em>Example:</em> <code>"test-suite"</code>
+    </td>
+   </tr>
+  <tr>
+    <td><code>env</code></td>
+    <td>
+      A map of <a href="/docs/pipelines/environment-variables">environment variables</a> for this step.<br>
+      <em>Example:</em> <code>RAILS_ENV: "test"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>if</code></td>
+    <td>
+      A conditional to whether or not this group should be included in the build. A boolean expression that omits the step when false. See <a href="/docs/pipelines/conditionals">Using conditionals</a> for supported expressions.<br>
+      <em>Example:</em> <code>build.message != "skip me"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>key</code></td>
+    <td>
+    	A unique string to identify the block step. Alias: `identifier` or `id`.
+      <em>Example:</em> <code>"test-suite"</code>
+    </td>
+  </tr>  
+  <tr>
+    <td><code>label</code></td>
+    <td>
+      The label that will be displayed in the pipeline visualisation in Buildkite (name of the group in the UI). Supports emoji.<br>
+      <em>Example:</em> <code>"\:hammer\: Tests" will be rendered as "🔨: Tests"</code><br>
+    </td>
+  </tr>
+  <tr>
+    <td><code>skip</code></td>
```

Comment:
File this under "needs better docs" I reckon. Something for another time.

Reply:
The `label` attribute or attributes in general? @plaindocs

Reply:
Skipping builds with the `skip attribute`